backend/routes/upload_reservation_routes.py

```
@@ -46,18 +46,46 @@ def upload_file():
+        required_fields = ['reservationId', 'name', 'surname', 'birthday', 'street', 'number_city',
+                           'cap', 'telephone', 'document_type', 'document_number', 'cf']
+
+        # Portale Alloggi required fields
+        portale_required_fields = ['sesso', 'nazionalita', 'email', 'comune_nascita',
+                                  'provincia_nascita', 'stato_nascita', 'cittadinanza',
+                                  'luogo_emissione', 'data_emissione', 'data_scadenza',
+                                  'autorita_rilascio', 'comune_residenza', 'provincia_residenza', 'stato_residenza']
 
         # Check if required files are in request
         if any(file_key not in request.files for file_key in required_files):
             raise BadRequest("Missing one or more required image files")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)